readme.md

```diff
@@ -82,26 +82,42 @@ Also, **for now**, there is only the `pathfinder` module, `movements` and `goals
 
 # Functions:
 
-`bot.pathfinder.bestHarvestTool(block)`
+bot.pathfinder.bestHarvestTool(block)
+`block` - Block instance
 
-`bot.pathfinder.getPathTo(movements, goal, done, timeout)`
+bot.pathfinder.getPathTo(movements, goal, done, timeout)
+`Return` - Path instance
+`movments` - Movements instance
+`goal` - Goal instance
+`done` - ?
+`timeout` - ?
```

Comment:
The maximum number of milliseconds to spend calculating a path before stopping.

Reply:
I Applied that description on bot.pathfinder.thinkTimeout
is that ok?